.env.example

```diff
@@ -3,3 +3,6 @@ DATABASE_URL='DATABASE_URL'
 EMAIL='email@mail.com
 CLIENT_ID=''
 TOKEN_ISSUER=''
+CLD_CLOUD_NAME=''
+CLD_CLOUDINARY_API_KEY=''
+CLD_CLOUDINARY_SECRET=''
```

Comment:
Just curious, why `CLD_` as prefix to the names? 🤔

Reply:
```CLOUDINARY```

Reply:
maybe just do it as `CLOUDINARY_CLOUD_NAME`, `CLOUDINARY_API_KEY` , etc ...